cmd/CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ if(NOT SILKWORM_CORE_ONLY)
   add_executable(sentry "sentry.cpp;${COMMON_SRC}")
   target_link_libraries(sentry PRIVATE silkworm_node silkworm-buildinfo CLI11::CLI11 silkworm_sentry)
 
+  add_executable(sentry_client_test "sentry_client_test.cpp")
+  target_link_libraries(sentry_client_test PRIVATE silkworm_sentry)
+
```

Comment:
I suggest to move this `sentry_client_test` in the `cmd/test` folder where other test clients are already present (e.g. `backend_kv_test`)

Reply:
👍